uaclient/util.py

```diff
@@ -336,9 +334,9 @@ def readurl(
     debug_msg = "URL [{}] response: {}, headers: {{{}}}, data: {}".format(
         method or "GET", url, sorted_header_str, content
     )
-    if potentially_sensitive:
-        # For large responses, this is very slow (several minutes)
-        debug_msg = redact_sensitive_logs(debug_msg)
+    # if potentially_sensitive:
+    #     # For large responses, this is very slow (several minutes)
+    #     debug_msg = redact_sensitive_logs(debug_msg)
```

Comment:
I think we need a solution to this in this PR.

I believe `potentially_sensitive` is only set to `False` in one place - it defaults to true.

We can change the semantics of this to get a similar result though. The only place where this is set to `False` doesn't really need the full response to be logged anyway.

So maybe remove the `potentially_sensitive` argument, and add a new `log_response_body` argument that defaults to `True`. Then here we'd have something like:
```python
    debug_msg = "URL [{}] response: {}, headers: {{{}}}".format(
        method or "GET", url, sorted_header_str
    )
    if log_response_body:
        # For large responses, this is very slow (several minutes)
        debug_msg += ", data: {}".format(content)
    logging.debug(debug_msg)
```